src/api/spec/factories/notification.rb

```diff
@@ -1,8 +1,8 @@
 FactoryBot.define do
   factory :notification do
-    event_type 'FakeEventType'
-    event_payload(fake: 'payload')
-    subscription_receiver_role 'owner'
+    event_type { 'FakeEventType' }
+    event_payload { { fake: 'payload' } }
```

Comment:
Nitpick: Couldn't it be `event_payload { fake: 'payload' }`?

Were the cops fixed with autocorrect or manually?

What I have done for including new cops is to create a commit for the corrections of each cop.

Reply:
This was fixed with autocorrect

Reply:
I will split the commits per cop.